### PR TITLE
fix: resolved issue with builder folder

### PR DESCRIPTION
**Pull request by janrtvld**
Signed-off-by: Jan <jan@animo.id>

### CODE REVIEW

File: server/tsconfig.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "../tsconfig.build.json"
+  "extends": "../tsconfig.build.json",
+  "compilerOptions": {
+    "outDir": "./build"
```
Comment (berendsliedrecht) [inline server/tsconfig.json:4]: Wouldn't it make sense to add this to the `tsconfig.build.json` as I think we would want the `build` folder to exist for every package that could be built. (We wouldn't want to use dist somewhere and build somewhere else).

**Review by berendsliedrecht — APPROVED** [review]
LGTM

File: server/tsconfig.json (hunk 1 above)
Comment (janrtvld) [reply server/tsconfig.json:4]: Adding this line to the `tsconfig.build.json` resulted in a build without the src directory:
image

Adding it to the `tsconfig.json` in the `server` directory resulted in a correct build. 

Do you have any idea how this happens?

Comment (berendsliedrecht) [reply server/tsconfig.json:4]: No, not really. We can leave it as is.

**Pull request merged**